layers/meta-tegrademo/conf/machine/jetson-tx2-devkit-cot.conf

```diff
@@ -3,9 +3,9 @@ require conf/machine/jetson-tx2-devkit.conf
 PACKAGE_EXTRA_ARCHS_append = " jetson-tx2-devkit"
 KERNEL_CLASSES += "tegra-kernel-fitimage"
 KERNEL_IMAGETYPE = "fitImage"
-TEGRA_ESSENTIAL_EXTRA_RDEPENDS_remove = "u-boot-extlinux"
+TEGRA_ESSENTIAL_EXTRA_RDEPENDS_remove = "kernel-image u-boot-extlinux"
 TEGRA_ESSENTIAL_EXTRA_RDEPENDS_append = " tegra-fitimage"
-PREFERRED_PROVIDER_virtual/bootloader = "u-boot-tegra"
```

Comment:
You sure you want to remove this? I think you'll still need it for non-Mender builds.

Reply:
Yes, I'm sure, I tested it. 

* if you want to use `Jetson TX2 COT machine`, you will have the `u-boot` as default bootloader.

```
$ bitbake -e demo-image-base | grep -i "^PREFERRED_PROVIDER_virtual/bootloader"
PREFERRED_PROVIDER_virtual/bootloader="u-boot-tegra"
```